src/app/src/main/resources/application-dev.properties

```diff
@@ -0,0 +1 @@
+spring.datasource.jdbc-url=jdbc:postgresql://localhost/hint
```

Comment:
Allows us to run the app outside docker against a dockerised db mapping to local port 5432